elements/lisk-p2p/src/peer_directory/peer_book.ts

```diff
@@ -31,18 +42,34 @@ export class PeerBook {
 		triedPeerConfig,
 		secret,
 	}: PeerBookConfig) {
-		this._newPeers = new NewPeers(newPeerConfig ? newPeerConfig : { secret });
+		this._newPeers = new NewPeers(
+			newPeerConfig
+				? newPeerConfig
+				: {
+						secret,
+						peerBucketCount: DEFAULT_NEW_BUCKET_COUNT,
+						peerBucketSize: DEFAULT_NEW_BUCKET_SIZE,
+						peerType: PEER_TYPE.NEW_PEER,
+				  },
+		);
 		this._triedPeers = new TriedPeers(
-			triedPeerConfig ? triedPeerConfig : { secret },
+			triedPeerConfig
+				? triedPeerConfig
+				: {
+						secret,
+						peerBucketCount: DEFAULT_TRIED_BUCKET_LENGTH,
+						peerBucketSize: DEFAULT_TRIED_BUCKET_SIZE,
+						peerType: PEER_TYPE.TRIED_PEER,
+				  },
 		);
 	}
 
 	public get newPeers(): ReadonlyArray<P2PPeerInfo> {
-		return this._newPeers.newPeersList();
+		return this._newPeers.peersList();
 	}
 
 	public get triedPeers(): ReadonlyArray<P2PDiscoveredPeerInfo> {
-		return this._triedPeers.triedPeersList();
+		return this._triedPeers.peersList() as ReadonlyArray<P2PDiscoveredPeerInfo>;
 	}
 
```

Comment:
not from this PR but imo it would be nicer if the order of listed functions after `getAllPeers` below is: `getPeer`, `updatePeer`, `addPeer`, `removePeer`, `upgradePeer`, `downgradePeer`.

Reply:
@ishantiw I'd make it in this PR. Why not? It is straightforward change.

Reply:
Oh I've made the changes in this PR already